lib/my_movie/my_movie_screen.dart

```diff
@@ -57,43 +61,33 @@ class _MyMovieScreenState extends State<MyMovieScreen> {
         centerTitle: true,
         title: const Text(_kTitle),
       ),
-      body: BlocConsumer<MyMovieCubit, MyMovieState>(
-        listener: (context, state) {},
+      floatingActionButton: Align(
+        alignment: Alignment.bottomRight,
+        child: FloatingActionButton(
+          onPressed: () {
+            Navigator.push(
+              context,
+              AddMovieScreen.route(myMovieCubit),
+            );
+          },
+          backgroundColor: AppColors.darkBlueBackground,
+          child: const Icon(Icons.add),
+        ),
+      ),
+      body: BlocBuilder<MyMovieCubit, MyMovieState>(
         builder: (context, state) {
-          if (state.status == MyMovieStatus.success) {}
           return SafeArea(
```

Comment:
add param `bottom: false` to resolve this problem
<img width="418" alt="image" src="https://user-images.githubusercontent.com/20111049/168553036-83a65809-cbea-4556-a7da-bf67691d7996.png">